src/renderer/components/layout/sidebar.tsx

```diff
@@ -98,108 +97,100 @@ export class Sidebar extends React.Component<Props> {
           </div>
           <div className="sidebar-nav flex column box grow-fixed">
             <SidebarNavItem
-              id="cluster"
```

Comment:
I think the integration tests rely on these ids

Reply:
@ixrock pls revert removal of ids.

Reply:
This is bad.

Reply:
yes it's bad, but let's fix that in separate PR